apps/src/music/views/instructions.module.scss

```diff
@@ -81,51 +82,50 @@
     }
 
     .text {
-      flex: 1;
       overflow-y: auto;
       white-space: pre-wrap;
+      flex: 1;
+      color:  $neutral_light;
+    }
 
-      &.textVertical {
-        padding: 0 10px;
-        margin: 10px 0;
+    .markdownText {
+      line-height: 0;
+
+      h1, h2 {
```

Comment:
I'm guessing we need this for the markdown styling? Can't use classnames instead?

Reply:
Unfortunately this is all we get from the `#` and `##` syntax being used.